pilot/pkg/config/kube/gateway/testdata/backend-tls-policy.yaml.golden

```diff
@@ -0,0 +1,80 @@
+apiVersion: networking.istio.io/v1
+kind: DestinationRule
+metadata:
+  annotations:
+    internal.istio.io/parents: BackendTLSPolicy/default.tls-upstream-echo
+  creationTimestamp: null
+  name: echo-istio-autogenerated-k8s-gateway
+  namespace: default
+spec:
+  host: echo.default.svc.domain.suffix
+  trafficPolicy:
+    tls:
+      credentialName: configmap://default/auth-cert
```

Comment:
this is a nice new feature- we should include that in the docstring in `istio/api` and add a separate release note

Reply:
added. thanks!